src/Websites/Websites/Utilities/WebsitesClient.cs

```diff
@@ -780,8 +781,20 @@ public IList<Snapshot> GetSiteSnapshots(string resourceGroupName, string webSite
             }
         }
 
-        public IList<DeletedSite> GetDeletedSites()
+        public IList<DeletedSite> GetDeletedSitesFromLocations(IEnumerable<string> locations)
         {
+            List<Task<IPage<DeletedSite>>> tasks = new List<Task<IPage<DeletedSite>>>();
+            foreach (string location in locations)
+            {
+                tasks.Add(WrappedWebsitesClient.DeletedWebApps.ListByLocationAsync(location));
```

Comment:
@Nking92 should we follow the `nextLink` returned from each of these operations, or are we only concerned about getting the first page for each location?

Reply:
Good point, but internally we don't page the response, so this code should technically be ok for now. The first page will have all the sites.